docs/src/starknet/account-import.md

```diff
@@ -136,4 +136,17 @@ $ sncast \
     --address 0x1 \
     --private-key 0x2 \
     --type oz
+```
+
+#### Auto-generated Account Name
+
+To import an account with an auto-generated name, skip the `--name` flag.
+
+```shell
+$ sncast \
+    account import \
+    --url http://127.0.0.1:5050 \
+    --address 0x1 \
+    --private-key 0x2 \
+    --type oz
 ```
```

Comment:
Instead of this, I'd, under the shell in `#### General Example`, add something like this "If the `--name` is not provided a default name will be generated instead.